packages/@lwc/template-compiler/src/parser/parser.ts

```diff
@@ -73,7 +75,7 @@ interface SiblingScope {
 }
 
 export default class ParserCtx {
-    private readonly source: String;
```

Comment:
Not seeing any references to this field outside of this class. Can this still be `private`?

Reply:
Good catch. This was necessary in a previous revision, but the need has been removed.